src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -255,6 +275,45 @@ private function resolveQueryOriginFromBacktrace(): ?string
         return "{$filePath}:{$firstAppFrame->getLine()}";
     }
 
+    protected function redisCommandExecutedHandler(RedisEvents\CommandExecuted $event): void
+    {
+        if (!$this->traceRedisCommands) {
+            return;
+        }
+
+        $parentSpan = SentrySdk::getCurrentHub()->getSpan();
+
+        // If there is no tracing span active there is no need to handle the event
+        if ($parentSpan === null) {
+            return;
+        }
+
+        // @TODO: Do we need/want to prefix all data with `db.redis`? It's already in the `db.redis` op so maybe it's not needed?
+        $data = [
+            'db.redis.connection' => $event->connectionName,
+            // @TODO: Sending all the parameters should possibly be locked behind `send_default_pii`? It does contain sensitive data but also Redis integration is opt-in so maybe it's fine?
+            'db.redis.parameters' => $event->parameters,
```

Comment:
The Ruby SDK locks this behind `send_default_pii` which makes sense... what is common sense here?

Reply:
Let's gate it behind `send_default_pii` and/or make it configurable as with SQL params.